### ISSUE TITLE
data_loader.load() can not give the result

### ISSUE THREAD

**Issue by ZFgan** [microsoft/qlib#421]
## ❓ Questions and Help
When I run the following code repeatedly, sometimes I can't give the result.

MACD_EXP   = '(EMA($close, 12) - EMA($close, 26))/$close - EMA((EMA($close, 12) - EMA($close, 26))/$close, 9)/$close'
fields = [MACD_EXP]
fiels_names= ['MACD']
labels = ['Ref($close,-2)/Ref($close,-1)-1']
label_names = ['LABEL']
data_handler_config = {
    'feature':(fields,fiels_names),
    'label':(labels,label_names)
}
data_loader = QlibDataLoader(config=data_handler_config)
df = data_loader.load(instruments='csi300',start_time = '2015-01-01',end_time = '2017-12-31')

first time is  6s
second time is 6s
third time can not give the result  ，It's running all the time.

**Comment by zhupr** [comment]
Can you provide the full code for the whole process, I can't reproduce it

Run the following code to get the environment information: `python scripts/collect_info.py all`
